terasoluna-gfw-functionaltest-web/src/main/resources/ValidationMessages.properties

```diff
@@ -1,3 +1,7 @@
 # this file contains the default messages of JSR 303 bean validation for validations provided in the common library
 
-org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
\ No newline at end of file
+org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...